taf/testlib/virtual_env.py

```diff
@@ -28,7 +28,9 @@
 #  "instance_type": "settings",
 #  "id": "993",
 #  "images_share_path": "/mnt/berta/oses/openstack",
-#  "mgmt_ip_cidr": "11.212.23.11/8",
+#  "external_net_ip_cidr": "11.212.23.0/24",
```

Comment:
external_net_ip_cidr was renamed to external_net_gw_ip_cidr as it have to contain gateway IP in cidr notation.
New example value:  # "external_net_gw_ip_cidr": "192.168.31.1/24"